src/Parser.js

```diff
@@ -98,6 +98,13 @@ export function parse (value, root, parent, rule, rules, rulesets, pseudo, point
 									case 100: case 109: case 115:
 										parse(value, reference, reference, rule && append(ruleset(value, reference, reference, 0, 0, rules, points, type, rules, props = [], length), children), rules, children, length, points, rule ? props : children)
 										break
+									// c
```

Comment:
```diff
+ switch (atrule === 99 && charat(characters, 3) === 110 ? 100 : atrule) {
- switch (atrule) {
```

Reply:
Is this a change you'd like me to make? Not sure if a commit or other suggestion has gone missing here. Also I'm not sure if there's a benchmark or the like you'd like me to run beforehand? Thanks for reviewing!

Reply:
Yes, this, instead of the current implementation, this one uses the least bytes and branches.